### DISCUSSION TITLE
Interested in porting to Emacs?

### DISCUSSION THREAD

**Discussion by kwkarlwang** [ggandor/lightspeed.nvim#44] [General]
Hi, first of all I want to say this is an amazing plugin! I consider this to be an essential plugin in my workflow now. Not really an issue but since you have implemented this in Lisp, I was wondering if you are interested in implementing this plugin in Emacs? Evil-snipe and Avy are substitutes for vim-sneak and EasyMotion, but I don't think there is anything like lightspeed in Emacs.

**Comment by ggandor** [comment]
> I don't think there is anything like lightspeed in Emacs

Well, I guess Neovim users will enjoy some bragging rights for a while :)

Hi, and thanks for your words! Porting to Emacs would be a good idea of course, and if someone would start the work, I'd be happy to contribute and gladly give all the help that is needed. That said, right now I have enough stuff on my todo list, and ideas about new/enhanced features, to keep me occupied for a year or so. Also, I'm not particularly familiar with the Emacs API, so I'd definitely need help with that. But if anyone is interested, just reach out and let's get to it.
